campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -225,7 +225,7 @@
         {{ donationAmountFourth }} {{ currencySymbol|default("$", true) }}
       </button>
 
-      <a class="button button-red button-wide donate-button  {{ buttonAnimate|default("", true) }}" target="_blank" href="#">
+      <a class="button button-red button-wide donate-button  {{ buttonAnimate|default("", true) }}" href="#">
```

Comment:
btw we are now expecting templates to be valid XML so this would raise an error while saving. I changed it to ` {% if buttonAnimate %} {{ buttonAnimate }} {% endif %}`